modules/apps/journal/journal-web/test/js/modals/ImportDataDefinitionModal.js

```diff
@@ -0,0 +1,212 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import '@testing-library/jest-dom/extend-expect';
+import {act, fireEvent, render, waitForElement} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+
+import Modal from '../../../src/main/resources/META-INF/resources/js/modals/ImportDataDefinitionModal';
+
+const ImportDataDefinitionModal = () => <Modal portletNamespace="test" />;
+
+describe('Import Structure Modal', () => {
+	let container, file;
+
+	beforeAll(() => {
+		Liferay.component = jest.fn();
+		container = render(<ImportDataDefinitionModal />);
+		file = new File(['testing file upload'], 'Testing.lar', {
+			type: '.lar',
+		});
+	});
+
+	it('renders the title modal', async () => {
+		const {getByText} = container;
+
+		act(() => {
+			Liferay.component.mock.calls[0][1].open();
+		});
+
+		const title = await waitForElement(() => getByText('import-structure'));
+
+		expect(title).toBeInTheDocument(true);
+	});
+
+	it('renders the info alert', () => {
+		const {queryByText} = container;
+
+		const infoAlert = queryByText(
+			'the-import-process-will-run-in-the-background-and-may-take-a-few-minutes'
+		);
+
+		expect(infoAlert).toBeInTheDocument(true);
```

Comment:
[toBeInTheDocument()](https://github.com/testing-library/jest-dom#tobeinthedocument) needs no parameters:
```suggestion
		expect(infoAlert).toBeInTheDocument();
```